src/Microsoft.Extensions.ProjectModel.Sources/MsBuild/MsBuildProjectContext.cs

```diff
@@ -16,15 +17,18 @@ internal class MsBuildProjectContext : IProjectContext
         private const string EmbedItemName = "EmbeddedResource";
         private const string FullPathMetadataName = "FullPath";
 
-        private readonly ProjectInstance _project;
-        private readonly string _name;
+        protected readonly ProjectInstance _project;
```

Comment:
Can you rename to match our conventions for protected properties? 
` protected ProjectInstance Project { get; }`